docs/DeveloperGuide.md

```diff
@@ -1,13 +1,39 @@
 # Developer Guide
 
+* Table of Contents
```

Comment:
The table of contents header is present but left blank. Perhaps filling it up would make it better so users could have an overview of what to expect in your Developer Guide!